### ISSUE TITLE
Fedora: rp-stub has the wrong path for KDE5

### ISSUE THREAD

**Issue by GerbilSoft** [GerbilSoft/rom-properties#126]
On Fedora, rp-stub has the wrong path for the KDE5 plugin:
```
/usr/plugins/rom-properties-kde5.so
/usr/lib64/thunarx-2/rom-properties-xfce.so
/usr/lib64/nautilus/extensions-3.0/rom-properties-gnome.so
```
Not sure if the other two are correct, so those will need to be checked as well.

**Comment by GerbilSoft** [comment]
This seems to be similar to #69 (QT_PLUGIN_INSTALL_DIR path problem), but I thought I fixed that before...

**Comment by GerbilSoft** [comment]
Fixed by b6b4d73914aa29f569b3553febb0297d1b4feb6e.
